roles/ipareplica/library/ipareplica_prepare.py

```diff
@@ -601,10 +602,14 @@ def main():
     ansible_log.debug("-- CA_CRT --")
 
     cafile = paths.IPA_CA_CRT
-    if not os.path.isfile(cafile):
-        ansible_module.fail_json(
-            msg="CA cert file is not available! Please reinstall"
-            "the client and try again.")
+    if is_ipa_client_configured(on_master=True):
```

Comment:
This should be executed only if `install_ca_cert` is not available anymore.

Reply:
Hopefully fixed.